x-pack/legacy/plugins/siem/public/components/timeline/body/column_headers/column_header.tsx

```diff
@@ -16,29 +18,10 @@ import { Sort } from '../sort';
 import { DraggingContainer } from './common/dragging_container';
 
 import { Header } from './header';
-import { ColumnId } from '../column_id';
-
-export type ColumnHeaderType = 'not-filtered' | 'text-filter';
-
-/** The specification of a column header */
-export interface ColumnHeader {
-  aggregatable?: boolean;
-  category?: string;
-  columnHeaderType: ColumnHeaderType;
-  description?: string;
-  example?: string;
-  format?: string;
-  id: ColumnId;
-  label?: string;
-  linkField?: string;
-  placeholder?: string;
-  type?: string;
-  width: number;
-}
 
 interface ColumneHeaderProps {
   draggableIndex: number;
-  header: ColumnHeader;
+  header: IColumnHeader;
```

Comment:
I don't have anything against naming things with a capital I at the front, but just wonder why we are changing it here and there? Should we be moving towards I with interfaces?

If it's just subjective and per person that's fine. Just curious.

Reply:
@FrankHassanabad I had a similar question, but I think it's because this file also defines a const `ColumnHeader` ? One should be a type and the other a value so it might not be strictly needed, but I'm guessing this is just a disambiguation. My comment about changing this to `options: ColumnHeaderOptions` is related.

Reply:
You are right @rylnd but I agree, I will rename it to ColumnHeaderOptions, that makes more sense to me too.